cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -256,7 +256,7 @@ if(builtin_lz4)
     URL ${lcgpackages}/lz4-${lz4_version}.tar.gz
     URL_MD5 c9610c5ce97eb431dddddf0073d919b9
     INSTALL_DIR ${CMAKE_BINARY_DIR}
-    CONFIGURE_COMMAND  /bin/sh -c "PREFIX=<INSTALL_DIR> make cmake"
+    CONFIGURE_COMMAND  /bin/sh -c "PREFIX=<INSTALL_DIR> CMAKE_PARAMS='-DCMAKE_C_COMPILER=\\\"${CMAKE_C_COMPILER}\\\" -DCMAKE_C_FLAGS=\\\"${CMAKE_C_FLAGS}\\\" -DCMAKE_OSX_SYSROOT=\\\"${CMAKE_OSX_SYSROOT}\\\"' make cmake"
```

Comment:
Unfortunately, `LZ4` is not using `cmake` -- it is invoking `make`, which is internally invoking `cmake`.

If I recall correctly, `sh` had to be invoked directly in order to get the `PREFIX` env var set right.